packages/xmlcatmgr.rb

```diff
@@ -0,0 +1,19 @@
+require 'package'
+
+class Xmlcatmgr < Package
+  description 'XML and SGML catalog manager'
+  homepage 'https://xmlcatmgr.sourceforge.net'
+  version '2.2'
+  compatibility 'all'
+  source_url 'https://downloads.sourceforge.net/sourceforge/xmlcatmgr/xmlcatmgr-2.2.tar.gz'
+  source_sha256 'ea1142b6aef40fbd624fc3e2130cf10cf081b5fa88e5229c92b8f515779d6fdc'
+
+
+  def self.build
+      system "./configure #{CREW_OPTIONS} "
+      system "make -j#{CREW_NPROC}"
```

Comment:
`-j#{CREW_NPROC}` is useless as this value is already defined by default in package.rb